src/renderer/screens/exchange/Buy/SelectAccountAndCurrency.js

```diff
@@ -96,6 +96,65 @@ const SelectAccountAndCurrency = ({ selectAccount, defaultCurrency, defaultAccou
     dispatch(openModal("MODAL_ADD_ACCOUNTS", { currency }));
   }, [dispatch, currency]);
 
+  const addOrSelectAccount = useMemo(() => {
```

Comment:
Is there a specific reason for a `useMemo`? It seams like this is always re-rendered anyway considering the dependencies of this memoized component.

Alternatively, it could make sense to create a new `AddOrSelectAccount` or even 2 separated `AddAccount` and `SelectAccount` components to further separate concerns, leverage component composition and reduce components re render

Reply:
Nope you're right, it's a reminiscence of a feature I tried to add but that didn't work. Removed. 👍 

Keeping it in 1 variable was on purpose, mainly to avoid creating another variable that would be named "selectAccountSomething" that would be conflicting with already declared selectors with similar names, making it way more confusing.

(Concerning the separate concerns, as discussed in my other PR, there will be no useless derender/rerender because the React renderer will diff everything)